apps/asap-server/test/fixtures/users.fixtures.ts

```diff
@@ -6,97 +6,91 @@ import {
   FetchUsersQuery,
 } from '../../src/autogenerated-gql/graphql';
 
-export const getGraphqlResponseFetchUsers = (): {
-  data: FetchUsersQuery;
-} => ({
-  data: {
-    queryUsersContentsWithTotal: {
-      total: 2,
-      items: [
-        getGraphQLUser(),
-        {
-          id: 'user-id-2',
-          created: '2020-09-23T20:45:22Z',
-          lastModified: '2020-10-26T15:33:18Z',
-          version: 42,
-          flatData: {
-            biography: 'some biography',
-            institution: 'some institution',
-            jobTitle: 'some job title',
-            onboarded: true,
-            orcidLastModifiedDate: null,
-            orcidLastSyncDate: null,
-            orcidWorks: [
-              {
-                doi: 'test-doi',
-                id: '987-654-321',
-                lastModifiedDate: '2020-10-26T15:33:18Z',
-                publicationDate: {},
-                type: 'BOOK',
-                title: 'orcid work title 2',
-              },
-            ],
-            reachOut: 'some reach out',
-            responsibilities: 'some responsibilities',
-            researchInterests: 'some research interests',
-            city: 'some city',
-            country: 'some country',
-            contactEmail: 'some@contact.email',
-            degree: 'some degree',
-            orcid: 'orcid',
-            avatar: null,
-            email: 'iwillbeback@arnold.com',
-            firstName: 'Arnold',
-            lastModifiedDate: null,
-            lastName: 'Schwatzneger',
-            questions: [],
-            expertiseAndResourceTags: [],
-            expertiseAndResourceDescription: 'Amazing person',
-            social: [
-              {
-                github: 'awesome',
-                googleScholar: null,
-                linkedIn: null,
-                researcherId: null,
-                researchGate: null,
-                twitter: null,
-                website1: null,
-                website2: null,
-              },
-            ],
-            teams: [
+export const getSquidexUsersGraphqlResponse = (
+  { total = 1 } = { total: 1 },
+): FetchUsersQuery =>
+  generateGraphqlFetchUsersResponse(
+    [...Array(total)]
+      .fill(getGraphQLUser())
+      .map((_, index) => getGraphQLUser({ id: `user-id-${index + 1}` })),
+  );
+export const getSquidexUserGraphqlResponse = (): FetchUserQuery => ({
+  findUsersContent: getGraphQLUser(),
+});
+
+const generateGraphqlFetchUsersResponse = (
```

Comment:
I'd stick to the `get....` naming convention

Reply:
Named it differently as it's not an exported function. Its purpose is to be a helper function generating the boilerplate around the structure of the response.

Happy to change it, but wanted it to be different to the 2 methods being called to get the fixtures.

Let me know.